src/Foundatio.AzureServiceBus/Messaging/AzureServiceBusMessageBus.cs

```diff
@@ -151,14 +204,14 @@ public class AzureServiceBusMessageBus : MessageBusBase<AzureServiceBusMessageBu
             if (_options.TopicEnableExpress.HasValue)
                 td.EnableExpress = _options.TopicEnableExpress.Value;
 
-            if (!String.IsNullOrEmpty(_options.TopicUserMetadata))
-                td.UserMetadata = _options.TopicUserMetadata;
+            //if (!String.IsNullOrEmpty(_options.TopicUserMetadata))
+            //    td.UserMetadata = _options.TopicUserMetadata;
```

Comment:
If disabled because of issues in GH (like [this](https://github.com/Azure/azure-sdk-for-net/issues/3635) one), ideally would be good to see the issue URL in a comment.

Reply:
@shahbarkha https://github.com/Azure/azure-sdk-for-net/issues/3635 Any idea if this is now supported?

Reply:
@niemyjski Its still not supported!!